functions/src/__tests__/authentication.test.ts

```diff
@@ -35,7 +35,7 @@ describe("useAuthentication", () => {
   });
 
   it("throws when request is null", async () => {
-    const fn: any = jest.fn();
+    const fn: any = jest.fn(); // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I am surprised you are getting these given jest.fn() returns JestMock. You should be able to remove these.